pkg/process/init.go

```diff
@@ -173,6 +181,51 @@ func (p *Init) Create(ctx context.Context, r *CreateConfig) error {
 	return nil
 }
 
+func (p *Init) validateRuncFeatures(ctx context.Context, bundle string) error {
```

Comment:
PTAL this for inspecting runtime features:
- https://github.com/containerd/containerd/pull/8509

Reply:
I have a local branch based on that PR and using the RuntimeInfo to check it. But with [the latest Derek's comment](https://github.com/containerd/containerd/pull/8509#issuecomment-1709115818) I'll wait to see if that gets re-worked or not.

If we merge this now as it is, I can always do a follow-up PR to adjust it to that once that is merged (if it is merged without major changes, I already have it ready :))

Reply:
given the fluidity of the changes @dmcgowan and @AkihiroSuda are discussing @rata pls add a TODO to refactor this when a decision is made for 1) determining how best to identify requirements from the config.json for this feature, and 2) what the general pattern of use is going to be for inspection of runtime info/features.

Reply:
Added (one line below :))